src/api/app/assets/stylesheets/webui/bootstrap_variables/colors.scss

```diff
@@ -22,7 +21,6 @@ $warning: $yellow;
 $info: $cyan;
 $success: $green;
 
-$card-cap-bg: $gray-200;
```

Comment:
`$card-cap-bg` is still used here:
- https://github.com/openSUSE/open-build-service/blob/55a05a719d9db402c32f4b69f136afbe5cd7ad3e/src/api/app/assets/stylesheets/webui/cm2/bootstrap.scss#L58
- https://github.com/openSUSE/open-build-service/blob/55a05a719d9db402c32f4b69f136afbe5cd7ad3e/src/api/app/assets/stylesheets/webui/coderay.scss#L10

Should we remove, or at least adapt, those lines?